kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -296,6 +400,8 @@
         resetWorkingResourcePool,
         contentPresentInWorkingResourcePool,
         //contentList,
+        cannotSelectTopicCard$,
+        cannotSelectSomeTopicWarning$,
```

Comment:
I also want to note this string is showing at up at the first level when I  use back navigation to navigate back . In addition, I tested with QA channel and `tigil-fajod` but one will not show up at the end of the routes when using the back button to exit the navigation except the active working resource pool .
<img width="781" alt="Screenshot 2024-02-27 at 00 03 02" src="https://github.com/learningequality/kolibri/assets/103313919/19e90114-b9d1-4473-a75a-076e4a324465">

Reply:
I think that once #11862 is merged with this that this will be resolved - I'll make note of this to check it specifically in testing after merging.